consensus/istanbul/core/roundchange.go

```diff
@@ -32,79 +32,151 @@ func (c *core) sendNextRoundChange() {
 
 // sendRoundChange sends the ROUND CHANGE message with the given round
 func (c *core) sendRoundChange(round *big.Int) {
-	logger := c.logger.New("state", c.state)
+	logger := c.logger.New("state", c.state, "cur_round", c.current.Round(), "cur_seq", c.current.Sequence(), "func", "sendRoundChange", "target round", round)
 
 	cv := c.currentView()
 	if cv.Round.Cmp(round) >= 0 {
-		logger.Error("Cannot send out the round change", "current round", cv.Round, "target round", round)
+		logger.Error("Cannot send out the round change")
 		return
 	}
 
-	c.catchUpRound(&istanbul.View{
+	nextView := &istanbul.View{
```

Comment:
Why don't we need to `catchUpRound` here anymore?

Reply:
I've placed the responsibility of doing what `catchUpRound` did (bumping waiting round) with whoever calls `sendRoundChange`